include/vcpkg/base/expected.h

```diff
@@ -74,7 +74,8 @@ namespace vcpkg
                  std::enable_if_t<std::is_convertible_v<ConvToT, T> &&
                                       !std::is_same_v<std::remove_reference_t<ConvToT>, Error>,
                                   int> = 0>
-        ExpectedT(ConvToT&& t) : m_t(std::forward<ConvToT>(t)), value_is_error(false)
+        ExpectedT(ConvToT&& t) noexcept(std::is_nothrow_constructible_v<T, ConvToT>)
```

Comment:
Does this need to be `std::is_nothrow_constructible_v<T, ConvToT&&>` since it's getting forwarded?

Reply:
`ConvToT` and `ConvToT&&` are the same thing. :) https://github.com/microsoft/vcpkg-tool/pull/511#discussion_r883062761